src/style.css

```
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  --header-blue-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--header-blue-color);
+}
+
+.is-active {
+  color: var(--header-blue-color);
+}
```

```suggestion
.nav__link:hover,
.is-active {
  color: var(--header-blue-color);
}
```
you can join a few selectors with same properties